src/helpers/pathGenerator.js

```diff
@@ -0,0 +1 @@
+export default id => (id ? `activities/items/${id}` : 'activities/items');
```

Comment:
idea is good, but implementation not. This is not pathGenerator. This is a specific path generator for `activities/items`.